src/api/.rubocop_todo.yml

```diff
@@ -285,6 +285,7 @@ Metrics/ModuleLength:
   Exclude:
     - 'app/helpers/flag_helper.rb'
     - 'app/helpers/maintenance_helper.rb'
+    - 'app/helpers/webui/notification_helper.rb'
```

Comment:
I guess by now you can remove this again or?

Reply:
good catch